code/framework/auditing/src/main/java/io/cattle/platform/iaas/api/auditing/AuditLogsRequestHandler.java

```diff
@@ -0,0 +1,31 @@
+package io.cattle.platform.iaas.api.auditing;
+
+import io.cattle.platform.api.auth.Policy;
+import io.github.ibuildthecloud.gdapi.context.ApiContext;
+import io.github.ibuildthecloud.gdapi.model.Schema;
+import io.github.ibuildthecloud.gdapi.request.ApiRequest;
+import io.github.ibuildthecloud.gdapi.request.handler.AbstractApiRequestHandler;
+
+import java.io.IOException;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class AuditLogsRequestHandler extends AbstractApiRequestHandler {
+
+    private static final Logger log = LoggerFactory.getLogger(AuditLogsRequestHandler.class);
+    @Inject
+    AuditService auditService;
+
+    @Override
+    public void handle(ApiRequest request) throws IOException {
+        if (!Schema.Method.GET.isMethod(request.getMethod())){
+            request.setAttribute("requestEndTime", System.currentTimeMillis());
+            Policy policy = (Policy) ApiContext.getContext().getPolicy();
+            log.info("Logged ApiRequest from: " + policy.getUserName());
```

Comment:
Usernames are unique in a rancher instance. User1 is always user1.(Assuming auth types are never changed.)